ocs_ci/ocs/md_blow.py

```diff
@@ -0,0 +1,155 @@
+        Get current DB percentage
+
+        Args:
+            db_percentage (int): Percentage of DB to be monitored
+        """
+        current_db_percentage = self.noobaa_db_pod.exec_cmd_on_pod(
+            "df -h | grep postgresql | awk '{print $5}'",
+            shell=True,
+        )
+        current_db_usage = int(current_db_percentage.strip().replace("%", ""))
+        logger.info(current_db_usage)
+        if current_db_usage >= db_percentage:
+            self.stop_dumping.set()
+            logger.error(
+                f"Provided fill percentage is invalid. Given {db_percentage}... Current usage of DB {current_db_usage}"
+            )
+        if db_percentage >= 100:
+            logger.error(f"Invalid value. Given {db_percentage}")
+            self.stop_dumping.set()
+        count = 0
+        while not self.stop_dumping.is_set():
+            if current_db_usage >= db_percentage:
+                self.stop_dumping.set()
+                logger.info(f"DB is filled with {db_percentage}")
+            prev_db_percentage = self.noobaa_db_pod.exec_cmd_on_pod(
+                "df -h | grep postgresql | awk '{print $5}'",
+                shell=True,
+            )
+            prev_db_usage = int(prev_db_percentage.strip().replace("%", ""))
+            if prev_db_usage == current_db_percentage:
+                count += 1
+            else:
+                count = 0
+            if count == 100:
+                logger.error(
+                    "DB Percentage is stuck from last 500 seconds to the same value"
+                )
+                self.stop_dumping.set()
+            current_db_percentage = self.noobaa_db_pod.exec_cmd_on_pod(
+                "df -h | grep postgresql | awk '{print $5}'",
+                shell=True,
+            )
+            current_db_usage = int(current_db_percentage.strip().replace("%", ""))
+            sleep(5)
+        logger.info("Exiting from monitor task")
+
+    def upload_obj_using_md_blow(
+        self,
+        bucket_name="first.bucket",
+        db_percentage=None,
+        obj_count=1000,
+        concurrency=50,
+        chunks=200,
+        chunk_size=1,
+    ):
+        """
+        Upload objects directly into DB using MD Blow script present in core pod
+
+        Args:
+            bucket_name (str): Bucket name where objects need to be uploaded
+            db_percentage (int): Total percent of DB to be filled
+            obj_count (int): Number of objects to upload in given bucket
+            concurrency (int): Number of threads to be used while uploading data
+            chunks (int): Chunk numbers in each object
+            chunk_size (int): Chunk size in each object
+        """
+        base_cmd = (
+            "node /root/node_modules/noobaa-core/src/tools/md_blow.js --system=noobaa "
+            f"--email={self.email} "
+            f"--password={self.password} "
+            f"--bucket={bucket_name} "
+        )
+        if db_percentage is not None:
```

Comment:
We can avoid the giant indentation block by returning/failing early if the argument is invalid:
```suggestion
        if db_percentage is None:
            raise ValueError("db_percentage must be provided and in the range [0, 100].") 
```

Reply:
@sagihirshfeld We have else block which will run if user wants to dump IOs without considering `threshold_pct`